src/components/Header/Header.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Team } from "../../App";
+
+type HeaderProps = {
```

Comment:
```suggestion
interface HeaderProps {
```
Moze nie ma to super znaczenia ale do tej pory spotykałem się z preferencją interface nad type w każdym projekcie nad którym dłubałem
https://medium.com/raccoons-group/why-you-probably-shouldnt-use-react-fc-to-type-your-react-components-37ca1243dd13